Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Tools/Notifications.hs

```diff
@@ -835,6 +841,34 @@ sendPickupLocationChangedOverlay person req entityData = do
     notifTitle = FCMNotificationTitle $ fromMaybe "Title" req.title -- if nothing then anyways fcmShowNotification is false
     body = FCMNotificationBody $ fromMaybe "Description" req.description
 
+sendCancellationRateNudgeOverlay ::
+  ( CacheFlow m r,
+    EsqDBFlow m r
+  ) =>
+  Id DMOC.MerchantOperatingCity ->
+  Person ->
+  FCM.FCMNotificationType ->
+  FCM.FCMOverlayReq ->
+  CancellationRateBaseNudgeData ->
+  m ()
+sendCancellationRateNudgeOverlay mOpCityId person fcmType req entityData = do
+  transporterConfig <- findByMerchantOpCityId mOpCityId (Just (DriverId (cast person.id))) >>= fromMaybeM (TransporterConfigNotFound person.merchantOperatingCityId.getId)
+  FCM.notifyPersonWithPriority transporterConfig.fcmConfig (Just FCM.HIGH) (clearDeviceToken person.id) notificationData (FCMNotificationRecipient person.id.getId person.deviceToken) EulerHS.Prelude.id
+  where
+    notificationData =
+      FCM.FCMData
+        { fcmNotificationType = fcmType,
+          fcmShowNotification = FCM.DO_NOT_SHOW,
```

Comment:
Why DO_NOT_SHOW?

Reply:
This is overlay which frontend handles on their own